core/src/core_tests/workflow_tasks.rs

```diff
@@ -2518,8 +2518,9 @@ async fn post_terminal_commands_are_discarded() {
         mock_workflow_client(),
     );
     mh.completion_mock_fn = Some(Box::new(|c| {
-        // Only the complete execution command should actually be sent
-        assert_eq!(c.commands.len(), 1);
+        // The start timer and complete execution commands should be sent, in
+        // that order.
+        assert_eq!(c.commands.len(), 2);
```

Comment:
This should probably assert the actual command types as well while we're at it

Reply:
Done